Sources/SwiftFormatPrettyPrint/TokenStreamCreator.swift

```diff
@@ -364,6 +364,231 @@ private final class TokenStreamCreator: SyntaxVisitor {
     return false
   }
 
+  // MARK: - Control flow statement nodes
+
+  override func visit(_ node: IfStmtSyntax) {
+    before(node.ifKeyword, tokens: .open(.inconsistent, 3))
```

Comment:
We were going to play with changing this offset in a separate branch, correct?

Reply:
Yeah, this change is just updating the curly brace handling to be consistent with the other nodes to make sure they all handle the empty case and empty-but-for-a-comment case consistently.